content/blog/2021/support-unknown.md

```diff
@@ -0,0 +1,348 @@
+  /* so we need the negated test to be "true" */
+  /* "yes, we do not support container queries" */
+}
+```
+
+I opened an issue 
+with the CSS Working Group,
+where Oriol Brufau pointed out
+that the answer depends on
+[exactly how we ask the question][table].
+Specifically, we can get different results
+by adding/removing function arguments,
+or wrapping our query in parenthesis.
+
+[table]: https://github.com/w3c/csswg-drafts/issues/6175#issuecomment-826822088
+
+I've expanded on his table here
+to show the different results across browsers.
+You can see the live result from your current browser, 
+along with known results from the other major browsers.
+I'm using `value` and `fn()` as placeholders 
+for "unknown tests" --
+since neither of these are supported anywhere:
+
+<table id="unknown-support">
+  <thead>
+    <th><code>@supports</code>…</th>
+    <th>Live Result</th>
+    <th>Firefox</th>
+    <th>Chromium</th>
+    <th>WebKit</th>
+  </thead>
+  <tbody>
+    <tr class="value">
+      <th><code>not value</code></th>
+      <td class="live"></td>
+      <td class="false">false</td>
+      <td class="false">false</td>
+      <td class="false">false</td>
+    </tr>
+    <tr class="value-wrap">
+      <th><code>not (value)</code></th>
+      <td class="live"></td>
+      <td class="true">true</td>
+      <td class="true">true</td>
+      <td class="true">true</td>
+    </tr>
+    <tr class="empty-fn">
+      <th><code>not fn()</code></th>
+      <td class="live"></td>
+      <td class="true">true</td>
+      <td class="false">false</td>
+      <td class="false">false</td>
+    </tr>
+    <tr class="fn">
+      <th><code>not fn(value)</code></th>
+      <td class="live"></td>
+      <td class="true">true</td>
+      <td class="true">true</td>
+      <td class="false">false</td>
+    </tr>
+    <tr class="empty-wrap">
+      <th><code>not (fn())</code></th>
+      <td class="live"></td>
+      <td class="true">true</td>
+      <td class="false">false</td>
+      <td class="true">true</td>
+    </tr>
+    <tr class="wrap-fn">
+      <th><code>not (fn(value))</code></th>
+      <td class="live"></td>
+      <td class="true">true</td>
+      <td class="true">true</td>
+      <td class="true">true</td>
+    </tr>
+  </tbody>
+</table>
+
+Again,
+what we want to see is `true` across the board.
+That would mean we can add future support for
+unknown features (e.g. `value` and `fn()`),
+and have old browsers acknowledge their lack of support.
+
+Earlier this week, 
+the CSS Working Group resolved that
+[unknown `@supports` expressions
+should evaluate to false 
+for all `@supports` rules][resolution] --
+which is good news moving forward.
+But the table above
+suggests that we also have a temporary solution
+we can use right away.
+
+[resolution]: https://github.com/w3c/csswg-drafts/issues/6175#issuecomment-839937577
+
+## The conclusion
+
+While we wait for browsers to standardize
+how they handle "negation of unknown support",
```

Comment:
```suggestion
how they handle "negation of unknown support,"
```

@stacyk 😂

Reply:
Counter proposal, to avoid the issue:

```suggestion
how they handle _negation of unknown support_,
```

("no political battles on company channels", or whatever…)